docker-compose.yml

```diff
@@ -1,19 +1,46 @@
 version: '3.7'
 
+x-defaults: &x-defaults
+  environment:
+    BINDING: 0.0.0.0
+    PORT: 3000
+    POSTGRES_HOST: "${POSTGRES_HOST:-db}"
+    POSTGRES_PORT: "${POSTGRES_PORT:-5432}"
+    SIGNAL_REST_CLI_ENDPOINT: "http://signal:8080"
+    SIGNAL_REST_CLI_ATTACHMENT_PATH: "signal-cli-config/attachments/"
+
 services:
   app:
+    <<: *x-defaults
+    command: bundle exec rails server
+    depends_on: [ db, queue ]
+
+  queue:
+    <<: *x-defaults
     environment:
       BINDING: 0.0.0.0
       PORT: 3000
       POSTGRES_HOST: "${POSTGRES_HOST:-db}"
       POSTGRES_PORT: "${POSTGRES_PORT:-5432}"
-    command: bundle exec rails server
-    depends_on: [ db ]
+      SIGNAL_REST_CLI_ENDPOINT: "http://signal:8080"
+      SIGNAL_REST_CLI_ATTACHMENT_PATH: "signal-cli-config/attachments/"
+    command: bundle exec rails jobs:work
+    depends_on: [ db, signal ]
 
   db:
     image: postgres:12.2-alpine
     volumes:
       - postgres_data:/var/lib/postgresql/data
 
+  signal:
+    image: bbernhard/signal-cli-rest-api:latest
```

Comment:
We should pin this to the current version and set up Dependabot updates for Docker.